cmd/cluster.go

```diff
@@ -287,6 +288,12 @@ func init() {
 	if err != nil {
 		log.Fatal(err.Error())
 	}
+	err = clusterCreateCmd.RegisterFlagCompletionFunc("audit", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		return []string{"off", "on", "splunk"}, cobra.ShellCompDirectiveNoFileComp
```

Comment:
You can add more specific documentation like this, which I think makes it nicer for the user:

```
		return []string{
		   "off\tdisables audit logging", 
		   "on\tdeploys...", 
		   "splunk\t..."
		 }, cobra.ShellCompDirectiveNoFileComp
```